Makefile

```diff
@@ -18,13 +18,18 @@ up-keycloak:
 	docker-compose -f docker-compose.yml -f docker-compose.keycloak.yml up \
 	--no-recreate --detach ${LIMIT}
 
+up-ipaipatrust:
```

Comment:
This is something we can test in PR CI, so I think we can start second IPA with just `make up`.  What do you think?

Reply:
Sure, PR updated.